### ISSUE TITLE
[FileUploader] Review accessibility

### ISSUE THREAD

**Issue by diogofscmariano** [lumada-design/hv-uikit-react#1292]
**Description**
The file upload, upon its development, included accessibility rules in the input (dropzone) and navigation with keyboard throughout its file list. Although, the elements in the file list do not include the correct html semantic elements not informs the user when an upload fails, succeeds or the messages displayed when this happens. 

Some useful links:
. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role
- https://scottaohara.github.io/a11y_styled_form_controls/src/meter/
- https://scottaohara.github.io/a11y_styled_form_controls/src/file-upload/
- https://benmarshall.me/styling-file-inputs/

**Acceptance Criteria**
- Make sure the File upload includes the correct semantic values and report accessible information about the files

**Notes**
- Review accessibility overall (is the input file described by the max file size and accepted files? does the user known what file he's removing when clicking a X button?);
- Add the progressbar role (and corresponding values) to the file progress status;
- Announce file upload success and fails (via a hidden aria-live area).
